portfolio/src/main/java/com/google/sps/comment/repository/impl/DatastoreCommentRepository.java

```diff
@@ -30,6 +31,7 @@ private Entity getCommentEntity(Comment comment) {
         Entity commentEntity = new Entity(ENTITY_NAME);
         commentEntity.setProperty(MESSAGE_PROPERTY, comment.getMessage());
         commentEntity.setProperty(SENDER_ID_PROPERTY, comment.getSenderId());
+        commentEntity.setProperty(IMG_URL_PROPERTY, comment.getImgUrl());
```

Comment:
4 spaces indentation? I think we need to pick if we want 2 or 4 spaces.

Reply:
From what I can see, it's constant 4. I'm not sure about what I need to change here.